api-reference/docstubs/api/driveitem-delete.md

```diff
@@ -0,0 +1,90 @@
+---
+title: "Delete driveItem"
+description: "Deletes a driveItem."
+author: ""
+localization_priority: Normal
+ms.prod: ""
+doc_type: apiPageType
+---
+
+# Delete driveItem
+
+Namespace: microsoft.graph
+
+Deletes a [driveItem](../resources/driveitem.md).
+
+## Permissions
+One of the following permissions is required to call this API. To learn more, including how to choose permissions, see [Permissions](/concepts/permissions-reference.md).
+
+|Permission type|Permissions (from most to least privileged)|
+|:---|:---|
+|Delegated (work or school account)|**TODO: Determine scopes **|
+|Delegated (personal Microsoft account)|Not supported.|
+|Application|**TODO: Determine AppOnly scopes **|
+
+## HTTP request
+<!-- {
+  "blockType": "ignored"
+}
+-->
+``` http
```

Comment:
While these might all be valid, this seems like some massive overgeneration.  Compare to: https://docs.microsoft.com/en-us/graph/api/driveitem-delete?view=graph-rest-beta&tabs=http